book/website/community-handbook/communication-channels/slack-start-guide.md

```diff
@@ -0,0 +1,110 @@
+# Quick Start Guide for Slack
+
+Welcome to the community Slack group: a place for you to network, collaborate, exchanges resources and share ideas with others in the community.
```

Comment:
Welcome to the community Slack group: a place for you to network, collaborate, exchange resources, and share ideas with others in the community.
```suggestion
Welcome to the community Slack group: a place for you to network, collaborate, exchanges resources and share ideas with others in the community.
```